scripts/aws_shakeout.py

```diff
@@ -0,0 +1,110 @@
+"""
+This script submits 100 jobs to spacer_test_jobs queue and monitors as the
+jobs are completed. The cluster is setup to add 20 instances as soon as we
+are jobs in the test_queue, so the 100 jobs are be completed quickly.
```

Comment:
I assume this should say "as soon as we **have** jobs in the test_queue, so the 100 jobs are ~be~ completed quickly"?

Reply:
yeah, have.